OJP/OJP_Trips.xsd

```diff
@@ -162,6 +167,54 @@
 					<xs:documentation>Whether the result should include operating day information - as encoded bit string and in natural language.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="HikingProfile" default="normal" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>Users walking profile (especially for hiking)</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="normal">
+							<xs:annotation>
+								<xs:documentation>e.g. yellow hiking signs</xs:documentation>
+							</xs:annotation>
+						</xs:enumeration>
+						<xs:enumeration value="medium">
+							<xs:annotation>
+								<xs:documentation>e.g. white-red-white hiking signs</xs:documentation>
+							</xs:annotation>
+						</xs:enumeration>
+						<xs:enumeration value="difficult">
+							<xs:annotation>
+								<xs:documentation>E.g. white-blue-white hiking signs</xs:documentation>
+							</xs:annotation>
+						</xs:enumeration>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="CyclingProfile" default="fast" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>Users cycling profile (especially for sportive activities).</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
```

Comment:
why "fast normal" and "green normal" in comments ? Is green the "normal" profile ?

Reply:
We will remove the "normal"s

Reply:
I suggest to switch annotation and enum values, since there are differences even in the alps:

https://akademie.alpinewelten.com/bergwandern/klassifizierung-von-wanderwegen#:~:text=Die%20Kategorie%20Blau,Sturzes%20drastische%20Konsequenzen%20haben%20w%C3%BCrden.

Let us use hard, medium and easy as values and describe the known color codes in the description.